client/jetpack-connect/signup.js

```diff
@@ -393,11 +393,22 @@ export class JetpackSignup extends Component {
 					</LoggedOutFormLinkItem>
 				);
 			} else {
-				header = wooDna.getServiceName();
+				const pluginName = wooDna.getServiceName();
+				header = pluginName;
 				if ( wooDna.getFlowName() === 'woodna:woocommerce-payments' ) {
 					subHeader = translate(
 						'Enter your email address to get started. Your account will enable you to start using the features and benefits offered by WooPayments'
 					);
+				} else if ( wooDna.getFlowName() === 'woodna:woo-blaze' ) {
+					/* translators: pluginName is the name of the Woo extension that initiated the connection flow */
+					subHeader = translate(
+						'Enter your email address to get started. Your account will enable you to start using the features and benefits offered by %(pluginName)s',
+						{
+							args: {
+								pluginName,
+							},
```

Comment:
Using this arg and not hardcoding the name "Blaze for WooCommerce". just so the main copy section can be reused across services if needed. _It's same as the one above it used by WooPayments and can be shared later if necessary (after translation)_